packages/protocol/test/identity/attestations.ts

```diff
@@ -609,34 +637,6 @@ contract('Attestations', (accounts: string[]) => {
       })
     })
 
-    describe('when an attestor has been authorized', () => {
```

Comment:
Why remove these tests?

Reply:
In the test setup, we set up an authorized attestation signer for every validator, thus this becomes superflous